src/Chapter06/Listing06.21.DefiningReadOnlyProperties.cs

```diff
@@ -1,20 +1,20 @@
 // Non-nullable field is uninitialized. Consider declaring as nullable.
 #pragma warning disable CS8618 // Pending a constructors
 
-namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_21
-{
+//namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_21
+//{
```

Comment:
Why commenting out this namespace?

Reply:
Because it is required when compiling with C# scripting. I'm trying to switch to Roslyn but that isn't complete.